lib/BaseLogger.js

```diff
@@ -56,21 +56,27 @@ class BaseLogger {
   }
 
   _writeLog(level, message) {
-    // We may have sent an Error;
-    // if we have, the stack is most useful.
-    if (message && message.stack) {
-      return this._writeLog(level, message.stack);
-    }
-
     const logData = {
       timestamp: moment.utc(),
       source: this.source,
       level,
       message
     };
+    // We may have sent an Error;
```

Comment:
I think we should move this into a separate file now.
Something like message mutators that can be called.
As we find more things want the logger to take care for us automatically we can apply it there.